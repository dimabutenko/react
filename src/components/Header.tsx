import React from 'react'
import './Header.module.scss'

const Header: React.FC = () => (
  <header className="card">
    <h1>TODO</h1>
  </header>
)

export default Header
