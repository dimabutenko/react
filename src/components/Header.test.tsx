import React from 'react'
import { render } from '@testing-library/react'
import Header from './Header'

describe('Header conpoonent tests', () => {
  test('Should render correctly', () => {
    const { container } = render(<Header />)
    expect(container).toMatchSnapshot()
  })
})
