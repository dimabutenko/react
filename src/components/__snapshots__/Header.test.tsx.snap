// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header conpoonent tests Should render correctly 1`] = `
<div>
  <header
    class="card"
  >
    <h1>
      TODO
    </h1>
  </header>
</div>
`;
